Jint/SourceParsedEventArgs.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Esprima.Ast;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Jint
+{
+    public sealed class SourceParsedEventArgs
+    {
+        public string SourceId { get; }
```

Comment:
nit: I think properties should come after constructors (fields before constructors), sorry for dropping notes but I seem to spot things in iterations, lazy brain or something..

Reply:
Nah, that's _my_ lazy brain - we've been through that convention already. 🙂